uaclient/contract.py

```diff
@@ -81,6 +82,17 @@ def request_contract_information(
         )
         return response_data
 
+    @util.retry(socket.timeout, retry_sleeps=[1, 3, 5])
+    def request_url(
+        self,
+        path,
+        **kwargs,
+    ):
+        return super().request_url(
+            path,
+            **kwargs,
+        )
```

Comment:
Python 3.5 is complaining about the syntax here... But we can oneline it and then Python sees the truth.
```suggestion
    def request_url(self, path, **kwargs):
        return super().request_url(path, **kwargs)
```

Reply:
In [this comment](https://github.com/canonical/ubuntu-advantage-client/pull/2171#discussion_r939159521), @lucasmoura and I agreed that the retry should not be applied to every `request_url` so I think this code will get removed anyway.